src/commands/dev/index.js

```diff
@@ -311,10 +317,15 @@ DevCommand.flags = {
     char: 'l',
     description: 'start a public live session',
   }),
+  edgeHandlers: flagsLib.boolean({
+    char: 'eh',
```

Comment:
it's common in the unix world to only have single-letter flags following a single-dash parameter so that you can combine multiple single letters in one param, e.g. `tar -xzf` which is short for `tar -x -z -f`. this kinda breaks with this convention and might leave users confused if there are isolated `-e` or `-h` (help?) flags?

multi-letter flags should always use double-dashes as a prefix imho